packages/react-components/src/components/RTTDisclosureBanner.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(rtt) */
+import { Icon, Link, Stack, Text, useTheme } from '@fluentui/react';
+/* @conditional-compile-remove(rtt) */
+import React from 'react';
+/* @conditional-compile-remove(rtt) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(rtt) */
+import {
+  bannerContainerStyles,
+  bannerContentStyles,
+  bannerLinkStyles,
+  bannerTitleClassName
+} from './styles/RTTDisclosureBanner.styles';
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * strings for rtt modal
+ */
+export interface RTTDisclosureBannerStrings {
+  bannerTitle?: string;
+  bannerContent?: string;
+  bannerLinkLabel?: string;
+}
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * Props for RTT Banner
+ */
+export interface RTTDisclosureBannerProps {
+  /** Link to learn more about RTT */
+  link?: string;
+  /** RTT Banner strings */
+  strings?: RTTDisclosureBannerStrings;
+}
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ * Banner to disclose that RTT is enabled for all participants for the entire duration of the meeting.
+ */
+export const RTTDisclosureBanner = (props: RTTDisclosureBannerProps): JSX.Element => {
+  const localeStrings = useLocale().strings.rttDisclosureBanner;
+  const strings = { ...localeStrings, ...props.strings };
+  const theme = useTheme();
+
+  return (
+    <Stack className={bannerContainerStyles()}>
```

Comment:
How come this doesn't reuse the notifications component we have?

Reply:
Is t his a banner on the top of composite or on top of captionBanner component?

Reply:
It's a banner inside the captions banner component

Reply:
I don't understand banner vs notification, why is this not another notification in the notification stack?

Reply:
<img width="542" alt="Screenshot 2024-10-25 at 9 27 08 AM" src="https://github.com/user-attachments/assets/70b0e4fa-a117-4236-b7a8-36ed0b0f310f">
This is the design, it's not a notification, it's a sticky banner inside captions banner. It's also of a different design versus the notification component, title, icon, and learn more link and box shadows are all different

Reply:
<img width="429" alt="Screenshot 2024-10-25 at 9 31 37 AM" src="https://github.com/user-attachments/assets/b88f8b1d-2c2d-45a8-8196-4ae6ba38d064">